engine/setup/src/include/general.cpp

```diff
@@ -228,38 +295,45 @@ void vs_itoa(char *line, int number, int length) {
 	return;
 }
 
-int pwer(int start, int end) {
+int pwer(int start, int end) 
+{
 	int current, val_return;
 	val_return = start;
-	for (current = 2; current <= end; current++) {
+	for (current = 2; current <= end; current++) 
+        {
 		val_return *= 10;
 	}
 	return val_return;
 }
 
-int pwr(int start, int end) {
+int pwr(int start, int end) 
+{
 	int current, val_return;
 	val_return = 1;
-	for (current = 2; current <= end; current++) {
+	for (current = 2; current <= end; current++) 
```

Comment:
Just FYI - `++current` is more efficient for this usage

Reply:
I have added in the requested changes, and tested the changes with modifying the vegastrilke.config file.